src/processor_arm.cpp

```diff
@@ -225,6 +231,8 @@ static constexpr CPUSpec<CPU, feature_sz> cpus[] = {
     {"cortex-a72", CPU::arm_cortex_a72, CPU::generic, 0, Feature::arm_cortex_a72},
     {"cortex-a73", CPU::arm_cortex_a73, CPU::generic, 0, Feature::arm_cortex_a73},
     {"cortex-a75", CPU::arm_cortex_a75, CPU::arm_cortex_a73, UINT32_MAX, Feature::arm_cortex_a75},
+    {"cortex-a76", CPU::arm_cortex_a76, CPU::arm_cortex_a73, UINT32_MAX, Feature::arm_cortex_a76}, // needs validation
+    {"neoverse-n1", CPU::arm_neoverse_n1, CPU::arm_cortex_a73, UINT32_MAX, Feature::arm_neoverse_n1}, // needs validation
```

Comment:
It wasn't clear to me what  the relation with `arm_cortex_a73` was on the a75 line that I copied. That might be the wrong value to carry forward

Reply:
It's the fallback CPU in case the LLVM version doesn't recognize this one.